### PR TITLE
Use existing aliases controlpanel permissions for aliases endpoint

### PR DESCRIPTION
**Pull request by jackahl**
fix for #1820

permissions are taken from https://github.com/plone/Products.CMFPlone/blob/f7f0b19197da7917db154cbb383bede97e1a90a7/Products/CMFPlone/controlpanel/permissions.zcml#L82

----
📚 Documentation preview 📚: https://plonerestapi--1825.org.readthedocs.build/

### CODE REVIEW

**Comment by jackahl** [comment]
@jenkins-plone-org please run jobs

File: src/plone/restapi/services/aliases/configure.zcml [hunk 1]
```diff
@@ -26,39 +26,39 @@
...
+      permission="Products.CMFPlone.ManageContextAliases"
       name="@aliases"
       />
 
   <plone:service
       method="DELETE"
       factory=".delete.AliasesDelete"
       for="Products.CMFPlone.interfaces.IPloneSiteRoot"
-      permission="cmf.ModifyPortalContent"
+      permission="Products.CMFPlone.ManageContextAliases"
```
Comment (davisagli) [inline src/plone/restapi/services/aliases/configure.zcml:61]: This one is on the site root so:

```suggestion
      permission="Products.CMFPlone.ManagePortalAliases"
```

**Review by davisagli — CHANGES_REQUESTED** [review]

**Comment by jackahl** [comment]
@jenkins-plone-org please run jobs

**Comment by jackahl** [comment]
@jenkins-plone-org please run jobs

**Review by davisagli — APPROVED** [review]
Thanks. The failing jenkins tests are unrelated. I'll merge.

**Pull request merged**